config.py

```diff
@@ -32,6 +32,7 @@ class Config(object):
     INVITE_EMAIL_FROM = 'enquiries@digitalmarketplace.service.gov.uk'
     INVITE_EMAIL_SUBJECT = 'Your Digital Marketplace invitation'
 
+    CREATE_USER_SUBJECT = 'Create your Digital Marketplace account'
```

Comment:
This seems like an odd thing to configurable. Seems to make more sense as a constant in the module.